guide/guide/setup.md

```diff
@@ -2,225 +2,90 @@
-
-</language>
-
-</branch>
-
-<language lang="en">
-
-::: warning
-You need to have discord.js version **atleast** v12
-:::
-
-</language>
-<language lang="tk">
+Below are all the available options for [`GCommandsClient`](https://gcommands.js.org/docs/#/docs/main/dev/typedef/GCommandsOptions):
+
+| PARAMETER             | REQUIRED | FUNCTIONALITY                                                                                                                                                                                                                                                                                          |
+| --------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| cmdDir                | ✅       | The directory to look for commands in                                                                                                                                                                                                                                                                  |
+| eventDir              | ❌       | The directory to look for events in                                                                                                                                                                                                                                                                    |
+| caseSensitiveCommands | ❌       | If set to true, command names will be case sensitive                                                                                                                                                                                                                                                   |
+| caseSensitivePrefixes | ❌       | If set to true, prefixes will be case sensitive                                                                                                                                                                                                                                                        |
+| unkownCommandMessage  | ❌       | If set to true, the bot will respond with an error message if a command the user tried to use doesn't exist                                                                                                                                                                                            |
+| language              | ✅       | The language the bot sends default messages in                                                                                                                                                                                                                                                         |

```

Comment:
If this is supposed to mean `the default language the bot uses`, I'd change it to
```suggestion
| language              | ✅       | The default language used to send messages                                                                                                                                                                                                                                                         |

```
Otherwise if there are some `default messages`, then it's probably correct?